src/pages/scheduler/hooks/useInitialDatetime.tsx

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from 'react';
+
+export const useInitialDateTime = (term?: string) => {
+  const today = useMemo(() => new Date(), []);
+  // determine what date to position the calendar on.
+  return useMemo<Date>(() => {
+    // if there is a term, use today as the default
+    if (term === undefined) return today;
```

Comment:
nit: does this work?
```suggestion
    if (!term) return today;
```

Reply:
looking back i'm pretty sure I wrote this code lol

Reply:
Yeah you did I just moved it lol